crates/sui/src/sui_commands.rs

```diff
@@ -260,6 +318,54 @@ impl SuiCommand {
                 }
                 Ok(())
             }
+            SuiCommand::Start {
+                config_dir,
+                force_regenesis,
+                #[cfg(feature = "indexer")]
```

Comment:
nit (optional): It may be helpful from a code organisation perspective to pull these params out into their own type, that gets flattened in, so you only need to feature gate a couple of things.

Reply:
Having seen more of the effects of feature gating, this nit is starting to feel less optional 😅

Reply:
That's a good idea, will do!